node/Dockerfile.in

```diff
@@ -0,0 +1,11 @@
+FROM ${NODE_IMAGE_NAME}:${NODE_IMAGE_VERSION}
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+USER root
+
+RUN npm install -g stylelint@${STYLELINT_VERSION} eslint@${ESLINT_VERSION}
```

Comment:
Since we don't do anything custom here, and only install eslint, maybe we could use a generic image that has eslint in the repos?

Reply:
Most node images start from Debian, Ubuntu or Alpine. Unfortunately none of them has styelint in the repos.